sdks/python/apache_beam/runners/portability/sdk_container_builder.py

```diff
@@ -338,3 +345,24 @@ def _get_gcs_bucket_and_name(gcs_location):
   def _make_tarfile(output_filename, source_dir):
     with tarfile.open(output_filename, "w:gz") as tar:
       tar.add(source_dir, arcname=SOURCE_FOLDER)
+
+  @staticmethod
+  def _get_cloud_build_machine_type_enum(machine_type: str):
+    if not machine_type:
+      return None
+    mappings = {
+        'n1-highcpu-8': cloudbuild.BuildOptions.MachineTypeValueValuesEnum.
```

Comment:
for my education, what is the default version?

Reply:
And could you remind me please when this becomes a problem?

Reply:
I've seen some docker exit code 137 error(seems to be OOM) when prebuilding with large dependencies such as tfx.